### PR TITLE
[Tests-Only] Moved isSidebarLinksTabPresent command into appSideBar

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Function `isSidebarTabPresent` and related methods/elements are moved into appSideBar PageObject

## Related Issue
- Part of #2677

## Motivation and Context
Appropriate placements for different PO methods and elements

## How Has This Been Tested?

- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js [hunk 1]
```diff
@@ -951,6 +921,10 @@ module.exports = {
     },
     filesTableHeader: {
       selector: '#files-table-header'
+    },
+    resourceDivSelector: {
```
Comment (individual-it) [inline tests/acceptance/pageObjects/FilesPageElement/filesList.js]: isn't that what we get from `getFileRowSelectorByFileName()`?

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js [hunk 2]
```diff
@@ -200,12 +201,31 @@ module.exports = {
...
+      const resourceDivXpath = util.format(
+        this.elements.resourceDivSelector.selector,
+        xpathHelper.buildXpathLiteral(resource)
+      )
+      this
+        .useXpath()
+        .click(resourceDivXpath)
+        .useCss()
+      return this.api.page.FilesPageElement.appSideBar()
+    },
```
Comment (individual-it) [inline tests/acceptance/pageObjects/FilesPageElement/filesList.js]: you should be able to use `clickRow()` instead, that should open the side bar

**Comment by individual-it** [comment]
one test fails
```
✖ Then group "grp1" should be listed as "Viewer" in the collaborators list for folder "simple-folder" on the webUI # tests/acceptance/stepDefinitions/sharingContext.js:764

Failed [strictEqual]: (displayed file name not as expected) - expected "simple-folder" but got: "[object Object]"
at NightwatchAPI.<anonymous> (/var/www/owncloud/phoenix/tests/acceptance/pageObjects/FilesPageElement/filesList.js:407:23)
at processTicksAndRejections (internal/process/task_queues.js:86:5)
```

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js (hunk 2 above)
Comment (kiranparajuli589) [reply tests/acceptance/pageObjects/FilesPageElement/filesList.js]: done!

File: tests/acceptance/pageObjects/FilesPageElement/filesList.js (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/pageObjects/FilesPageElement/filesList.js]: I've removed this element and use `clickRow` now...

**Pull request merged**